### PR TITLE
WEBUI-681: copy instance.clid to ftest server

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

**Review by semisse — APPROVED** [review]

**Review by semisse — DISMISSED** [review]

File: ftest/web-ui/webdriver/itests.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
       <artifact:file key="org.nuxeo.web.ui:nuxeo-web-ui-marketplace::zip" />
       <artifact:file key="org.nuxeo.web.ui:nuxeo-web-ui-marketplace-itests::zip" />
     </copy>
+    <copy todir="${nuxeo.home}/nxserver/data" file="instance.clid" />
```
Comment (Gabez0r) [inline ftest/web-ui/webdriver/itests.xml]: Do we already have an instance.clid file? 🤔

Comment (Gabez0r) [reply ftest/web-ui/webdriver/itests.xml]: Ah, I see this is related to https://github.com/nuxeo/nuxeo-web-ui/pull/1379. What happens if we run this locally and we don't have a instance.clid file?

Comment (richardsd) [reply ftest/web-ui/webdriver/itests.xml]: I added the instance clid in the pipeline https://github.com/nuxeo/nuxeo-web-ui/pull/1379/files 🤔  We shouldn't have the clid in the repository directly

Comment (richardsd) [reply ftest/web-ui/webdriver/itests.xml]: ~~If there's no clid, then it runs the tests without registering the server.~~ Build fails, so we must think about a strategy for this.

**Review by Gabez0r — DISMISSED** [review]

**Review by semisse — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**